db/migrate/20200409043946_create_classified_listing_categories.rb

```diff
@@ -0,0 +1,14 @@
+class CreateClassifiedListingCategories < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    create_table :classified_listing_categories do |t|
+      t.string :name, null: false
+      t.integer :cost, null: false
+      t.string :rules, null: false
+
+      t.timestamps
+    end
+    add_index(:classified_listing_categories, :name, unique: true, algorithm: :concurrently)
```

Comment:
no need for concurrently as it's an empty table :)

Reply:
The gem didn't let me do it.

Reply:
Ok, whatever the problem was was probably related to me doing/undoing/redoing these migrations a couple of times. I fixed it now.